src/content/docs/dropins/checkout/tutorials/add-payment-method.mdx

```diff
@@ -67,14 +72,14 @@ To integrate the Braintree payment provider with the Commerce boilerplate templa
    let braintreeInstance;
    ```
 
-1. Update the [`PaymentMethods`](/dropins/checkout/containers/payment-methods/) container to include a custom handler for the Braintree payment method and set `setOnChange` to `false` to prevent automatic calls to the [`setPaymentMethod`](/dropins/checkout/functions/#setpaymentmethod) function on change.
+1. Update the [`PaymentMethods`](/dropins/checkout/containers/payment-methods/) container to include a custom handler for the Braintree payment method and set `autoSync` to `false` to prevent automatic calls to the [`setPaymentMethod`](/dropins/checkout/functions/#setpaymentmethod) function on change.
```

Comment:
```suggestion
1. Update the [`PaymentMethods`](/dropins/checkout/containers/payment-methods/) container to include a custom handler for the Braintree payment method. Set `autoSync` to `false` to prevent automatic calls to the [`setPaymentMethod`](/dropins/checkout/functions/#setpaymentmethod) function when the payment method changes.
```